poll/public/css/poll.css

```diff
@@ -0,0 +1,218 @@
+/* CSS for PollBlock Student View */
+
+.poll-answer {
+    margin-left: 1em;
+    font-weight: bold;
+    display: inline-block;
+}
+.percentage-gauge {
+    display: inline-block;
+    background-color: #e5ebee;
+    position: relative;
+    z-index: 1;
+}
+
+.poll-result-input-container {
+    display: table-cell;
+    vertical-align: middle;
+}
+
+.poll-result-input-container input {
+    margin: 0 .5em 0 0;
+}
+
+.percentage-gauge-container {
+    display: table-cell;
+    width: 100%;
+    vertical-align: middle;
+    background-color: #fafbfc;
+}
+
+ul.poll-answers, ul.poll-answers-results {
+    list-style-type: none !important;
+    max-width: 80%;
+}
+
+li.poll-answer {
```

Comment:
@sarina `li.poll-answer` means "apply to `li` elements that have class `poll-answer`. `.po;;-answer li` means "apply to `li` elements that are *children* of any element that have class `poll-answer`. So it's not quite the same.